routes/web.php

```diff
@@ -90,3 +90,12 @@
 
 //フレンドの解除
 Route::post('mypage/friends/{id}/delete','UserController@friend_delete')->name('friend_delete');
+
+//通知ページ
+Route::get('mypage/notices','UserController@notices')->name('notices');
```

Comment:
恐らくUserControllerにあるからという意味で他のと合わせるために `mypage` を付けてくれたんだと思いますが、通知は独立したページなので下記のルーティングで良さそうです！

もしあれだったらControllerも専用の `NoticeController` を作成しても良いと思います
（専用のをつくるかどうかはお任せします！）
```suggestion
Route::get('notices','UserController@notices')->name('notices');
```